src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
```

Comment:
initialize inside the Animal class instead

Reply:
there is a remark 

![Screenshot_1](https://github.com/mate-academy/js_herbivores_and_carnivores/assets/119410378/58cff158-7bff-48ba-8dbb-1db9f04f6e66)